pycytominer/operations/noise_removal.py

```diff
@@ -43,7 +43,7 @@ def noise_removal(
     """
     # Subset dataframe
     if samples != "all":
-        population_df.query(samples, inplace=True)
+        population_df = population_df.query(samples)
```

Comment:
Seeing how these functions which implement a `query` pattern made me wonder if there's an opportunity in future changes to abstract the functionality into a decorator or class which can help provide the capability without the repetition. I mention this as a thought towards continuous improvement and not an expectation within this PR for review.